pcsx2/SPU2/Mixer.cpp

```diff
@@ -594,21 +594,6 @@ static __forceinline StereoOut32 MixVoice(uint coreidx, uint voiceidx)
 
 			switch (EmuConfig.SPU2.Interpolation)
 			{
-				case Pcsx2Config::SPU2Options::InterpolationMode::Nearest:
-					Value = GetVoiceValues<0>(thiscore, voiceidx);
-					break;
-				case Pcsx2Config::SPU2Options::InterpolationMode::Linear:
-					Value = GetVoiceValues<1>(thiscore, voiceidx);
-					break;
-				case Pcsx2Config::SPU2Options::InterpolationMode::Cubic:
-					Value = GetVoiceValues<2>(thiscore, voiceidx);
-					break;
-				case Pcsx2Config::SPU2Options::InterpolationMode::Hermite:
-					Value = GetVoiceValues<3>(thiscore, voiceidx);
-					break;
-				case Pcsx2Config::SPU2Options::InterpolationMode::CatmullRom:
-					Value = GetVoiceValues<4>(thiscore, voiceidx);
-					break;
 				case Pcsx2Config::SPU2Options::InterpolationMode::Gaussian:
```

Comment:
No need for the switch case anymore, code can be changed, maybe also remove the config itself for interpolation as there's no need.

Reply:
I'll mark as draft for now but I will remove the unnecessary bits as well.